web/css/codeworld.css

```diff
@@ -502,11 +507,11 @@ body {
     overflow: hidden;
     pointer-events: none;
     background: linear-gradient(to bottom,
-        rgba(0, 0, 0, 0.7) 0px,
-        rgba(0, 0, 0, 0.6) 2em,
-        rgba(0, 0, 0, 0) 3em,
-        rgba(0, 0, 0, 0) 100%);
-    color: white;
+        var(--obsolete-code-alert-grad-point1) 0px,
+        var(--obsolete-code-alert-grad-point2) 2em,
+        var(--obsolete-code-alert-grad-point3) 3em,
```

Comment:
The third and fourth color points will always be the same, and will always be 100% transparent.  You don't need variables for those.